build.sbt

```diff
@@ -1,23 +1,29 @@
-ThisBuild / tlBaseVersion                         := "0.43"
+ThisBuild / tlBaseVersion                         := "0.44"
 ThisBuild / tlCiReleaseBranches                   := Seq("master")
 ThisBuild / githubWorkflowEnv += "MUNIT_FLAKY_OK" -> "true"
-ThisBuild / scalacOptions += "-Xsource:3"
+
+ThisBuild / tlCiReleaseBranches += "topic/scala3"
 
 Global / concurrentRestrictions += Tags.limit(Tags.Compile, 1)
 
+ThisBuild / scalaVersion := "3.1.2"
+ThisBuild / crossScalaVersions := Seq("3.1.2")
+ThisBuild / scalacOptions += "-language:implicitConversions" // TODO
```

Comment:
It seems that the use of implicit conversions are becoming very discouraged, thus ideally we should remove this compiler flag in the long-term.

E.g. this is why I added the `.refined` syntax instead of auto-refinement.